CHANGELOG.md

```diff
@@ -0,0 +1,62 @@
+# Changelog
+All notable changes to this project will be documented in this file.
+
+The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
+and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
+
+
+## [Unreleased]
+
+### Added
+
+* Integrating APIs with security from [@ohltyler](https://github.com/ohltyler)
+
+### Changed 
+* Corrected ml/stats/nodes permissions spelling from [@camAtGitHub](https://github.com/camAtGitHub)
+* Internal user usage clarification from [@reshippie](https://github.com/reshippie)
+* Remove `advanced_modules_enabled` examples from [@wjordan](https://github.com/wjordan)
+* CHANGELOG.md replaces release-notes
+
+### Security
+
+* Security patch for `org.cryptacular:cryptacular`
+* Security patch for `org.springframework:spring-core`
+* Security patch for `org.apache.kafka:kafka-clients`
+* Security patch for `org.springframework.kafka:spring-kafka-test`
+
+## [2.0.0-rc1] - 2022-04-20
+
+### Added
+
+* DLS privileges evaluation can process security roles from [@ch-govau](https://github.com/ch-govau)
+* Setting to enable role without DLS/FLS to override roles
+* DLS Term Lookup Queries
```

Comment:
the changelog should contain enough information for users to understand what it means for them. e.g. here:

```suggestion
* Added support for Terms Lookup Queries in Document Level Security, see [the documentation](https://opensearch.org/docs/2.0/security-plugin/access-control/document-level-security/#use-term-level-lookup-queries-tlqs-with-dls) for further details on how to use this ([#1508](https://github.com/opensearch-project/security/issues/1508))
```